app/services/nsm/all_adjustments_deleter.rb

```diff
@@ -44,6 +45,15 @@ def delete_disbursement_adjustments
       end
     end
 
+    def delete_youth_court_fee_adjustment
+      revert(submission.data, 'include_youth_court_fee', 'additional_fees')
+      submission.data.delete('youth_court_fee_adjustment_comment')
+    end
+
+    def youth_court_fee
```

Comment:
Nit: This method is called `youth_court_fee` but doesn't return a youth court fee, it returns whether or not the youth court fee has been adjusted. (The other methods below it are badly named too, but less so, e.g. `letters_and_calls` returns letters and calls that have been adjusted, but at least they are letters and calls). Maybe we could call this `youth_court_fee_adjustment_comment`

Equally, because it's only used once, we don't need to memoize it.